pkg/component/istio/istio_test.go

```diff
@@ -2325,7 +2339,7 @@ spec:
 			It("should successfully deploy all resources", func() {
 				Expect(c.Get(ctx, client.ObjectKeyFromObject(managedResourceIstioSecret), managedResourceIstioSecret)).To(Succeed())
 				Expect(managedResourceIstioSecret.Type).To(Equal(corev1.SecretTypeOpaque))
-				Expect(managedResourceIstioSecret.Data).To(HaveLen(15))
+				Expect(managedResourceIstioSecret.Data).To(HaveLen(13))
```

Comment:
This is because of https://github.com/gardener/gardener/pull/8116/commits/8dfded8bf092fd4704cbe6c6683d204f1ab3105d
See https://github.com/kubernetes/kubernetes/issues/118588 for the reasoning behind it.